src/pybind/mgr/mgr_module.py

```diff
@@ -352,6 +352,10 @@ def CLIWriteCommand(prefix, args="", desc=""):
     return CLICommand(prefix, args, desc, "w")
 
 
+def get_localized_key(prefix, key):
```

Comment:
I'd call this `_get_localized_key`, as it's mostly called internally and users don't need it.

Reply:
Done